### PR TITLE
[docs] Add Pandas To Frameworks Section

### PR DESCRIPTION
**Pull request by JesterOrNot**
Add a Pandas Frameworks Guide! See #383

### CODE REVIEW

File: src/docs/frameworks/pandas.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Pandas in Gitpod
+
+Welcome data scientists! This guide will show you how to configure Gitpod for Pandas development!
+
+## Start Tasks
+
+Setting up [start tasks](/docs/config-start-tasks) for Pandas isn't very diffrent from other frameworks, but here is an example:
```
Comment (nisarhassan12) [inline src/docs/frameworks/pandas.md:7]: Maybe remove `, but` Is it necessary here :slightly_smiling_face: ?

File: src/docs/frameworks/pandas.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Try it
+
+To see a minimal project with Pandas installed and configured, please check out [gitpod-io/Gitpod-Pandas](https://github.com/gitpod-io/Gitpod-Pandas):
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/gitpod-io/Gitpod-Pandas)
+
+## Further Reading
+
+- [Python in Gitpod](/docs/languages/python) For further information on configuring Python projects in Gitpod
```
Comment (nisarhassan12) [inline src/docs/frameworks/pandas.md:25]: Not that important maybe put a dot at the end here :slightly_smiling_face: Is this without a dot grammatically valid if so then ignore this.

Comment (JesterOrNot) [reply src/docs/frameworks/pandas.md]: Good Catch!

File: src/docs/frameworks/pandas.md (hunk 1 above)
Comment (JesterOrNot) [reply src/docs/frameworks/pandas.md]: Done!

**Comment by nisarhassan12** [comment]
@JesterOrNot Please wait for @jankeromnes approval as well before merging this :slightly_smiling_face: .

**Comment by JesterOrNot** [comment]
Thanks, Merging!

**Pull request merged**